applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -29,7 +29,28 @@ void BaseSolidElement::Initialize()
 {
     KRATOS_TRY
 
-    const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints(  );
+    if( GetProperties().Has(INTEGRATION_ORDER) ) {
+        const SizeType integration_order = GetProperties()[INTEGRATION_ORDER];
+        switch ( integration_order )
+        {
+        case 1:
+            mThisIntegrationMethod = GeometryData::GI_GAUSS_1;
+        case 2:
+            mThisIntegrationMethod = GeometryData::GI_GAUSS_2;
+        case 3:
+            mThisIntegrationMethod = GeometryData::GI_GAUSS_3;
+        case 4:
+            mThisIntegrationMethod = GeometryData::GI_GAUSS_4;
+        case 5:
+            mThisIntegrationMethod = GeometryData::GI_GAUSS_5;
+        default:
+            mThisIntegrationMethod = GetGeometry().GetDefaultIntegrationMethod();
```

Comment:
Can you please add a warning here saying that the requested order is not available and the default one is used?

Reply:
Ok